osx/make-installer.sh

```diff
@@ -3,51 +3,51 @@
 # abort script if any command fails
 set -e
 
-# Retrieve latest source code, keep history as it's useful to edit sometimes
-if [ ! -d "source" ]; then
-  git clone https://github.com/Mudlet/Mudlet.git source
+# set path to find macdeployqt
+PATH=/usr/local/opt/qt5/bin:$PATH
+
+cd source/build
+
+# install installer dependencies
+brew update
+BREWS="sqlite3 lua@5.1 node wget"
+for i in $BREWS; do
+  brew outdated | grep -q $i && brew upgrade $i
+done
+for i in $BREWS; do
+  brew list | grep -q $i || brew install $i
+done
+if [ ! -f "macdeployqtfix.py" ]; then
+  wget https://raw.githubusercontent.com/aurelien-rainone/macdeployqtfix/master/macdeployqtfix.py
 fi
+echo "Need root permissions to install lua rocks..."
+sudo luarocks-5.1 install LuaFileSystem
+sudo luarocks-5.1 install lrexlib-pcre
+sudo luarocks-5.1 install LuaSQL-SQLite3 SQLITE_DIR=/usr/local/opt/sqlite
 
-# In case it already exists, update it
-cd source/
-git pull
-
-# Switch to release banrch
-git fetch
-git checkout release_30
-
-# Setup PATH to find qmake
-PATH=/Users/mudlet/Qt/5.8/clang_64/bin:$PATH
-
-cd src/
-# Remove old Mudlet.app, as macdeployqt doesn't like re-doing things otherwise. Requires admin rights.
-sudo rm -rf Mudlet.app/
-
-# Compile using all available cores
-qmake
-make -j `sysctl -n hw.ncpu`
+npm install -g appdmg
```

Comment:
On macOS < 10.12.3, https://github.com/LinusU/node-appdmg/issues/121 hits us - I had to patch ``appdmg`` with https://github.com/ArmorText/node-appdmg/commit/17dfeebdf01d42f519a77e6f43d4008cfb7dc3c7 to get the background to show.

Could you add a check for the version and apply the patch?